packages/react-component-library/src/components/RangeSlider/RangeSlider.test.tsx

```diff
@@ -156,30 +158,35 @@ describe('RangeSlider', () => {
       )
     })
 
-    it('should apply the stateful `is-disabled` class', () => {
-      expect(wrapper.queryByTestId('rangeslider').classList).toContain(
-        'is-disabled'
+    it('should apply the cursor `not-allowed` CSS property', () => {
+      expect(wrapper.queryByTestId('rangeslider')).toHaveStyleRule(
+        'cursor',
+        'not-allowed'
       )
     })
-  })
 
-  describe('reversed', () => {
-    beforeEach(() => {
-      wrapper = render(
-        <RangeSlider
-          mode={mode}
-          domain={domain}
-          values={values}
-          step={step}
-          isReversed
-        />
-      )
-    })
+    describe('and the user attempts to move the slider', () => {
+      beforeEach(() => {
+        fireEvent(
+          wrapper.getByTestId('rangeslider-handle'),
+          new MouseEvent('click', {
+            bubbles: true,
+            cancelable: true,
+          })
+        )
 
-    it('should apply the stateful `is-reversed` class', () => {
-      expect(wrapper.queryByTestId('rangeslider').classList).toContain(
-        'is-reversed'
-      )
+        fireEvent.keyDown(wrapper.getByTestId('rangeslider-handle'), {
```

Comment:
The `@testing-library/user-event` seems to make the use typing neater. Could be worth looking at? We have used it in `NumberInput`.

Reply:
I've changed the clicks to se userEvent but couldn't get the type API to play nice - seems to be more aimed at inputs:

https://github.com/testing-library/user-event#typeelement-text-options

Reply:
Okay fair enough. I now realise that the `userEvent.click` looks nice so I will use that in other places too.